src/cmd/apply.ts

```diff
@@ -0,0 +1,116 @@
+import { existsSync, mkdirSync, rmdirSync, writeFileSync } from 'fs'
+import { Argv, CommandModule } from 'yargs'
+import { $ } from 'zx'
+import { OtomiDebugger, terminal } from '../common/debug'
+import { giteaPush } from '../common/gitea-push'
+import { Arguments as HelmArgs, helmOptions } from '../common/helm-opts'
+import { hf, hfTrimmed, hfValues } from '../common/hf'
+import { ENV, LOG_LEVEL_STRING } from '../common/no-deps'
+import { cleanupHandler, otomi, PrepareEnvironmentOptions } from '../common/setup'
+import { decrypt } from './decrypt'
+import { Arguments as DroneArgs, genDrone } from './gen-drone'
+
+const fileName = 'apply'
+const dir = '/tmp/otomi/'
+const templateFile = `${dir}/deploy-template.yaml`
```

Comment:
To what does this file refer to? If I search in the whole repository I can't find it.

Reply:
It is an output path - as you can see, it uses `${dir}` that points to `/tmp/otomi`

Reply:
It emulates this functionality:
https://github.com/redkubes/otomi-core/blob/master/bin/deploy.sh#L7

It could possibly be rewritten to a pipe - which is becoming available due to my rewrite using streams. But not 100% sure